### PR TITLE
ipfs 0.14.0

### PR DESCRIPTION
**Pull request by Porkepix**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Porkepix** [comment]
Looking at your commit, should this formula be renamed, then?

**Comment by Moisan** [comment]
If the current name creates confusion we could always rename. Personally I would keep the status quo right now since upstream seems to use `kubo` and `ipfs` names interchangeably (from what I understand). 

If you or someone else decides to rename, you can follow our [documentation about renaming](https://docs.brew.sh/Rename-A-Formula). Also be aware that upstream points to the `ipfs` formula in their README.

**Comment by chenrui333** [comment]
I think we should probably rename this formula. ([upstream changelog reference](https://github.com/ipfs/kubo/pull/9112))

**Comment by SMillerDev** [comment]
Let's add an alias and we can always rename later.

File: Formula/ipfs.rb [hunk 1]
```diff
@@ -1,14 +1,14 @@
 class Ipfs < Formula
```
Comment (carlocab) [inline Formula/ipfs.rb:1]: Might be good to just rename this formula?

**Review by carlocab — DISMISSED** [review]

File: Formula/ipfs.rb (hunk 1 above)
Comment (carlocab) [reply Formula/ipfs.rb:1]: CC @chenrui333

**Comment by carlocab** [comment]
Oh, forgot `--no-autosquash`. Just as well, we probably want to do a formula rename here.

File: Formula/ipfs.rb (hunk 1 above)
Comment (carlocab) [reply Formula/ipfs.rb:1]: Wait, is the command still called `ipfs`? How confusing.

**Comment by chenrui333** [comment]
yeah, tbh, I am inclined to rename the formula.

**Comment by carlocab** [comment]
Actually, sorry. I changed my mind. The CLI tool is still called `ipfs`, so we can probably keep that for now. Sorry for the confusion!

**Review by carlocab — APPROVED** [review]
I reverted the name change, since the command is still called `ipfs`. The symlink seems better here.

@chenrui333 feel free to trigger the merge.

File: Formula/ipfs.rb (hunk 1 above)
Comment (Jorropo) [reply Formula/ipfs.rb:1]: We didn't wanted to break *too many things*.

AFAIT At first we wanted to rename and add `alias ipfs=kubo` to people's `.bashrc` but modifying peoples dot file like that is not a nice thing to do. (also there is far too much setups for us to get right everywhere)

So now it's still called `ipfs` and if want you can add `alias kubo=ipfs`.

I think an alias or symlink from `kubo` to `ipfs` would be nice.

**Comment by chenrui333** [comment]
Sounds alright, I think it already does cause some discrepancies between the package manager ([some repology ref](https://repology.org/project/kubo/versions)).

File: Formula/ipfs.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/ipfs.rb:1]: make sense, @Jorropo thanks for the input!

**Comment by chenrui333** [comment]
Thanks folks!

**Pull request closed without merging**